packages/block-library/src/table/index.js

```diff
@@ -1,30 +1,24 @@
 /**
  * External dependencies
  */
+
```

Comment:
I don't think we add spaces after dependencies section comment in other places. I don't think we have a rule for that, but sharing as an observation. Personally, I would let `prettier` to take care of formatting 😄